src/style.css

```diff
@@ -1,3 +1,87 @@
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+
+  --link-color: rgba(0, 172, 220, 1);
+}
+
+.header {
+  max-width: 1200px;
+  min-width: 1024px;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 1);
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  position: relative;
+  color: rgba(0, 0, 0, 1);
```

Comment:
no needs to use `rgba` since you don't add alpha-chanel. Either use `rgb` or `hex`